client/src/components/About.js

```diff
@@ -1,17 +1,17 @@
 import React from "react";
-
+import { Container, Header } from "semantic-ui-react";
 const About = () => {
   return (
-    <div>
-      About
+    <Container>
```

Comment:
You do not need to use Container here instead use ```React.Fragment``` because already you wrapped all contents in index.js so that will wrap all other components.

Reply:
.eg: ```<React.Fragment>content</React.Fragment>```

Reply:
done

Reply:
i just mentioned they are using it at any component